### DISCUSSION TITLE
The website broken directly after Enable it

### DISCUSSION THREAD

**Discussion by fadimoubassaleh** [rhubarbgroup/.github#53] [Support (ROC)]
Directly after enabling the cache it crash the website and shows this error
![image](https://github.com/rhubarbgroup/.github/assets/33605551/42957297-535f-46eb-80ea-29ce7c0efc7f)
any solution for that?

**Comment by tillkruss** [comment]
See https://github.com/rhubarbgroup/redis-cache/issues/526.
